CRFREE documented (#66)
crate/cloud-reference: docs/subscription-plans.rst

    different subscription plans. You can combine CrateDB Edge with any
    suitable hardware configuration that works for your use case.

Although you can scale your cluster from a single node to as many nodes as
desired, only clusters containing 3 or more nodes are covered by our 24/7
support. You can find our CrateDB Cloud terms and conditions in our `SLA`_.

CRFREE
------

This plan is aimed at new users that want to test and evaluate our offerings.
It is perpetually free to use. Every user can deploy one Free Tier cluster in

> their organization, without the need of adding a payment method. This plan 

Reviewer: ```suggestion
their organization without having to add a payment method. This plan
```

also doesn't consume any 
:ref:`Free Credit <cloud-tutorials:free-trial-budget>` that you may have
available. They are limited to one node with 2 CPUs, 2 GiB of memory, and 4
GiB of storage.

CRFREE was made for active testing and evaluation, which is why any CRFREE
cluster deletes after 7 days of inactivity. When this happens, the user is
notified via email.

CR0
---

> The CR0 plan, or Shared-resource cluster are a special category of clusters

Reviewer: ```suggestion
The CR0 plan, or Shared-resource cluster is a special category of cluster
```

> that allow better utilization of resources. These clusters share compute and 

Reviewer: ```suggestion
that allows better utilization of resources. These clusters share compute and
```

storage resources with other clusters in this category. Because of this, they 
offer a more cost-effective solution for smaller teams and experimental 
deployments of low-traffic applications. They are limited to one node with 2 
CPUs, 2 GiB of memory, and 4 GiB of storage.

CR1-CR4
-------

These are our base plans that offer the best performance, and the biggest
flexibility. Their specs start from 1.75 CPU, 7 GiB RAM, and 32GiB storage all
the way to 14 CPU, 55 GiB RAM, and 512 GiB storage per-node. These plans can 

> be to up to 9 nodes. If you need more nodes than this, feel free to 

Reviewer: ```suggestion
be scaled to up to 9 nodes. If you need more nodes than this, feel free to
```
